src/main/java/org/csanchez/jenkins/plugins/kubernetes/ContainerEnvVar.java

```diff
@@ -16,6 +16,8 @@
 @Deprecated
 public class ContainerEnvVar extends KeyValueEnvVar {
 
+    private static final long serialVersionUID = 42L;
+
```

Comment:
Useless for this kind of object that get serialized through XStream exclusively, not using java serialization.

Reply:
In Jenkins, yes.